apps/frontend/src/app/3_organisms/ZeroLocForm/hooks/useZeroData.ts

```diff
@@ -48,7 +49,7 @@ export const useZeroData = (rbtcPrice?: number): ZeroDataResponse => {
   return useMemo(
     () => ({
       ...response,
-      isRecoveryMode: response.tcr <= CRITICAL_COLLATERAL_RATIO,
+      isRecoveryMode: response.tcr.lte(CRITICAL_COLLATERAL_RATIO), // todo: check if it's correct
```

Comment:
What about this todo comment, have you checked it or who should do it?

Reply:
yep, todo comment should have been removed